src/firejail/profile.c

```diff
@@ -1789,6 +1789,17 @@ void profile_read(const char *fname) {
 			free(ptr);
 			ptr = tmp;
 		}
+		else if (strncmp(ptr, "whitelist-ro ", 13) == 0) {
```

Comment:
```C
while {
    …
}
else if
```

???

Anyway I can not review code changes in firejail.

Reply:
I think that's just github being weird when rendering:

https://github.com/netblue30/firejail/blob/894bf81a595304b0c1a843c85a607003fd419d1b/src/firejail/profile.c#L1785-L1802